### PR TITLE
Add new parameter SteeringWheelLocation

### PR DESCRIPTION
**Pull request by dtrunov**
Add new parameter SteeringWheelLocation to hmiCapabilities struct in RegisterAppInterface response

Related CRQ [APPLINK-23406](https://adc.luxoft.com/jira/browse/APPLINK-23406)

### CODE REVIEW

**Comment by dtrunov** [comment]
@LuxoftAKutsan @TMelnyk @okozlovlux please review

**Comment by TMelnyk** [comment]
reviewed

File: src/components/application_manager/include/application_manager/hmi_capabilities_impl.h [hunk 1]
```diff
@@ -378,34 +378,20 @@ class HMICapabilitiesImpl : public HMICapabilities {
       const smart_objects::SmartObject& prerecorded_speech) OVERRIDE;
 
   /*
-   * @brief Interface used to store information if navigation
+   * @brief Interface used to store information about ui hmi capabilities
    * supported by the system
-   *
-   * @param supported Indicates if navigation supported by the system
+   * @param ui_hmi_capabilities contains ui hmi capabilities
+   * supported by the sustem
```
Comment (okozlovlux) [inline src/components/application_manager/include/application_manager/hmi_capabilities_impl.h]: @dtrunov , `system` typo

**Comment by dtrunov** [comment]
Rebuild required

**Comment by dtrunov** [comment]
@@LuxoftAKutsan @okozlovlux please review latest changes

File: src/components/application_manager/test/request_tracker_test.cc [hunk 2]
```diff
@@ -211,7 +208,7 @@ TEST_F(RequestTrackerTestClass,
 TEST_F(RequestTrackerTestClass,
        TrackAppRequestInNone_DoPause_TrackAgain_ExpectSuccessTillLimit) {
   const uint32_t max_requests = 5;
-  const uint32_t time_scale_ms = 1;
+  const uint32_t time_scale_ms = 10;
```
Comment (okozlovlux) [inline src/components/application_manager/test/request_tracker_test.cc]: @dtrunov ,  `10u`

**Pull request merged**